Index: Makefile.in
===================================================================
--- Makefile.in	(revision 749)
+++ Makefile.in	(working copy)
@@ -9,7 +9,7 @@
 EXPAT_CFLAGS = @EXPAT_CFLAGS@
 ERLANG_CFLAGS= @ERLANG_CFLAGS@
 
-EXPAT_LIBS = @EXPAT_LIBS@
+EXPAT_LIBS = @EXPAT_LIBS@ -lotr
 ERLANG_LIBS = @ERLANG_LIBS@
 
 # make debug=true to compile Erlang module with debug informations.
@@ -28,7 +28,7 @@
 prefix = @prefix@
 
 SUBDIRS = @mod_irc@ @mod_pubsub@ @mod_muc@ @mod_proxy65@ @eldap@ @web@ stringprep @tls@ @odbc@ @ejabberd_zlib@
-ERLSHLIBS = expat_erl.so
+ERLSHLIBS = expat_erl.so otr_drv.so
 SOURCES = $(wildcard *.erl)
 BEAMS = $(SOURCES:.erl=.beam)
 
